portfolio/src/main/webapp/script.js

```diff
@@ -22,8 +22,19 @@ async function getComments() {
   const commentsElement = document.getElementById('comments-container');
   commentsElement.innerHTML = '';
 
+<<<<<<< HEAD
+  for (var comment of comments) {
+    commentsElement.appendChild(createCommentElement(comment));
+  }
+
+  if (commentsElement.innerHTML === '') {
+    document.getElementById("comment-section").style.visibility="hidden";
+  } else {
+    document.getElementById("comment-section").style.visibility="visible";
+=======
   for (const comment of comments) {
     commentsElement.appendChild(createCommentElement(comment))
+>>>>>>> master
```

Comment:
Seems like there's some merge artifacts?

Reply:
Oops, I didn't realize this happened. Done.